externals/kyuubi-hive-sql-engine/pom.xml

```diff
@@ -147,7 +147,13 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>2.6.1</version>
```

Comment:
Extract it as a maven property and manage it under `<dependencyManagement>` block in root pom before using.

Reply:
done